### DISCUSSION TITLE
Advisor nie usuwa rekomendacji po ich wdrożeniu

### DISCUSSION THREAD

**Discussion by kmadof** [AzureCommunityPL/group#114] [Pomoc] [answered]
Znalazłem kilka rekomendacji w Azure Advisor i je zaaplikowałem. W zależności od środowiska robiłem to na dwa sposoby:

- automatem z poziomu Advisora (żeby przy okazji upewnić się co on tam dokładnie zrobi)
- oraz manualnie

Ani jedno ani drugie nie znika mi z Advisora, a minęło już więcej niż 24h.

Problem ten tyczy się:

- Diagnostic logs in Azure Stream Analytics should be enabled
- Diagnostic logs in Event Hub should be enabled
- Auditing on SQL server should be enabled

Nie bardzo wiem jak to rozwiązać. Sprawdziłem z kolei Log Workspace i trafaiają tam logi. Ktoś może spotkał się z tym?

**Comment by kamil-mrzyglod** [comment, marked as answer]
Więcej tzn. ile? ;) Mi potrafiły te rekomendacje wisieć nawet i 3 dni po rozwiązaniu.

**Reply by kmadof** [reply]
Na czas pisania tego posta to było jakieś 28 godzin. Rozumiem, uzbroję się w cierpliwość. Zwiodło mnie to

![image](https://user-images.githubusercontent.com/1680172/95980005-d0ce8280-0e1c-11eb-81cd-1b86597f9b95.png)

**Reply by mifurm** [reply]
Niestety, MS formalnie nie daje żadnej gwarancji na czas przeliczenia... kiedyś o to prosiłem, nie dostałem nic konkretnego i nie można nic założyć.

**Reply by kmadof** [reply]
Dzięki za info. Na szczęście już zaczęło to znikać :) Aczkolwiek jest to jak najbardziej irytujące.
